test/test_nist.py

```diff
@@ -39,3 +39,26 @@ def test_inputTemperature(self):
         reference_temperature = get_reference_NIST_values(serial_number)
 
         np.testing.assert_array_almost_equal(test_temperature, reference_temperature, decimal=5)
+        
+    @pytest.mark.unit
+    def test_formatArray_version1(self):
+        temperature = 20
+        serial_number = 41
+
+        test_array = temperature_correction(temperature, serial_number)
+        
+        reference_temperature = get_reference_NIST_values(serial_number)
+
+        assert np.shape(test_array) == np.shape(reference_temperature)
+        assert isinstance(test_array,np.ndarray) == True
+
+    @pytest.mark.unit
+    def test_inputTemperature_version1(self):
+        temperature = 20
+        serial_number = 41
+
+        test_temperature = temperature_correction(temperature, serial_number)
+
+        reference_temperature = get_reference_NIST_values(serial_number)
+
+        np.testing.assert_array_almost_equal(test_temperature, reference_temperature, decimal=5)
```

Comment:
When I downloaded the file, there was actually a lot of empty lines after this. Have you heard of a Python linter? It cleans up your code for you and makes it closer to a standard PEP8 convention. Running `black` is a good one (but don't do this on all the code, just this file, otherwise there's going to be too many changes). See: https://github.com/psf/black

Reply:
I tried `black test_nist.py` and saw slight formatting changes. Let me know if this is ok for you